content/basics/vimmodes/normal.md

```diff
@@ -4,4 +4,10 @@ date =  2017-10-30T10:48:19-04:00
 weight = 5
 +++
 
-Normal Mode
+## Normal Mode  
+To enter in this mode, you need to *type* the **ESC** key.   
+This mode is where you will navigate in the file and you will can execute keys such dd(to remove the line on under the cursor, for example).
+And you will do tasks such *copy,paste,cut,find/replace,etc...* we will see it in the next steps.
```

Comment:
You can also perform tasks such as *copy, paste, cut, find/replace, etc...* We will see how in the following lessons.